cmd/kusk/cmd/generate.go

```diff
@@ -66,11 +67,25 @@ var generateCmd = &cobra.Command{
 				errors.NewErrorReporter(cmd, err).Report()
 			}
 		}
+		parsedApiSpec := &openapi3.T{}
+		overlay, err := overlays.NewOverlay(apiSpecPath)
+		if err == nil {
+			overlayPath, err := overlay.Apply()
+			if err != nil {
+				return err
+			}
 
-		parsedApiSpec, err := spec.NewParser(&openapi3.Loader{IsExternalRefsAllowed: true}).Parse(apiSpecPath)
-		if err != nil {
-			reportError(err)
-			return err
+			parsedApiSpec, err = spec.NewParser(&openapi3.Loader{IsExternalRefsAllowed: true}).Parse(overlayPath)
```

Comment:
`spec.NewParser(&openapi3.Loader{IsExternalRefsAllowed: true})` is repeated below.

Please see earlier comment.

Reply:
as above